charts/hush-sensor/templates/_helpers.tpl

```diff
@@ -103,6 +103,21 @@ type: Unconfined
     {{- end -}}
 {{- end }}
 
+{{/*
+Hush deployment info
+*/}}
+{{- define "hush-sensor.deploymentInfo" -}}
+{{- $parts := .Values.deploymentToken | b64dec | split ":" -}}
+{{- $zone := ternary "" (printf "%s." .Values.hushZone) (not .Values.hushZone) }}
+{{- $uri := printf "https://events.%s.%shush-security.com/v1/runtime-events" $parts._1 $zone -}}
+{{- $result := dict
+    "orgId" $parts._2
+    "deploymentId" $parts._3
+    "eventReportingUri" $uri
+-}}
+{{- $result | toYaml }}
```

Comment:
Do we really need `toYaml` here? Have you tried without it?
Also, do we need a closing `-}}` here?

Reply:
Returning a dict didn't seem to work - got an error about being unable to index a string.
https://stackoverflow.com/a/71172610 suggested that to/from json/yaml conversions seem to do the trick.